src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");

```

Comment:
The message in the `System.out.println` should be more informative. Mention that it is a Truck that has started working, to provide clarity in the output.